src/vconsole/vconsole-setup.c

```diff
@@ -470,24 +482,14 @@ static void setup_remaining_vcs(int src_fd, unsigned src_idx, bool utf8) {
                 if (cfo.op != KD_FONT_OP_SET)
                         continue;
 
+                if (verify_vc_kdmode(fd_d) < 0) {
+                        log_debug("KD_FONT_OP_SET skipped: tty%u is not in text mode", i);
```

Comment:
please use log_debug_errno() and pass error code as first argument, even if you don't output it via %m. our log functions generated structured logs if possible and will include the error numer in a structured log field if provided. hence: *always* pass the error code to the log functions if you have one, regardless of using %m or not